content/articles/getting-started-ssl-certificates.markdown

```diff
@@ -18,42 +18,47 @@ An [SSL certificate](/articles/ssl-certificates) is required to enable SSL on yo
 
-## SSL certificate requirements {#requirements}
+## Before you start {#requirements}
 
 Before starting, take a moment to review how you will use the certificate and be sure to have access to the server configuration, or a feature to install a custom certificate on your server.
 
 If you are not familiar with SSL certificates, you may want to learn more about the different [SSL certificate types](/articles/ssl-certificates-types).
 
-Here are some questions you should be able to answer:
+In some limited instances, you may need to use a [custom CSR](/articles/what-is-csr) to acquire a new certificate. Check your web server's documentation to determine if you do.
 
-- We [offer different types of certificates](/articles/ssl-certificates). Which type of certificate do you want to purchase?
-- What is the [appropriate host name](/articles/ssl-certificate-names) for your case?
-- Check the web server documentation to understand if you need a [custom CSR](/articles/what-is-csr).
+**You are not required to transfer or host a domain with us to request an SSL certificate**. You can acquire a Comodo SSL certificate for a domain, even if the domain is hosted elsewhere or registered with another registrar. Let's Encrypt certificates can be requested only for domains resolving with us. More details are available [on the articles about requesting the two types of certificates](#requesting).
 
-**You are not required to transfer or host a domain with us to purchase an SSL certificate**. You can purchase an SSL certificate for a domain, even if the domain is hosted elsewhere or registered with another registrar. Read the [ordering section](/articles/purchasing-ssl-certificates#order) of the article explaing SSL purchasing for more information.
+## Choose the certificate product
 
-Feel free to [contact us](https://dnsimple.com/contact) if you can't find the answer to any of the questions above, or if you have any doubt.
+We [offer different types of certificates](/articles/ssl-certificates). You will need to determine which version you would like to acquire.
 
-## Purchasing the SSL certificate {#purchase}
+## Choose the certificate names
 
-[Purchase](/articles/purchasing-ssl-certificates) the desired SSL certificate from your DNSimple control panel. If the SSL certificate is expiring and it was previously purchased with us, you can [renew](/articles/renewing-ssl-certificates) it and we will default some settings for you.
+Some types of certificates allow multiple names to be chosen for the certificate. Before requesting your certificate you will need to know what the [appropriate host name](/articles/ssl-certificate-names) is for your case.
 
-If you need to use a [custom CSR](/articles/what-is-csr), generate it and paste it in the SSL certificate purchase form.
+## Request the certificate {#requesting}
```

Comment:
I went with "Request" here, specifically because it has a meaning in SSL-land and what we're doing at this step is creating the actual request. Do you think we shouldn't use that language at all here?